client-src/elements/chromedash-drawer.ts

```diff
@@ -177,13 +174,13 @@ export class ChromedashDrawer extends LitElement {
       fetch('/dev/mock_login', {method: 'POST'})
         .then(response => {
           if (!response.ok) {
-            throw new Error('Sign in failed! Response:', response);
+            throw new Error('Sign in failed! Response: ${response.status}');
```

Comment:
This also needs backticks in order to be evaluated:
```suggestion
            throw new Error(`Sign in failed! Response: ${response.status}`);
```

Reply:
You missed this change.